### PR TITLE
rename util -> stdx

### PR DESCRIPTION
**Pull request by matklad**
Good thing its the same number of letters!

### CODE REVIEW

File: src/stdx.zig [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/// Extensions to the standard library -- things which could have been in std, but aren't.
```
Comment (sentientwaffle) [inline src/stdx.zig]: The `zig fmt` CI complains about this file. My guess is it wants this to be a module comment.
```suggestion
//! Extensions to the standard library -- things which could have been in std, but aren't.
```

Comment (matklad) [reply src/stdx.zig]: Argh, I was sure zig didn’t repeat Rust mistake of needing two types of the comments for this…

**Comment by sentientwaffle** [comment]
Now there's a git conflict :sweat_smile:

**Comment by matklad** [comment]
Yeah, this matklad always causes conflicts with my PRs!

But that’s for tomorrow me to resolve :)

**Comment by matklad** [comment]
Rebased, test failure seems to be a flake.

**Pull request merged**